src/openlcb/DatagramDefs.hxx

```diff
@@ -36,6 +36,10 @@
 
 #include <stdint.h>
 
+#if defined(ESP32)
+#undef DISPLAY
```

Comment:
Is there a better way to address this namespace pollution?  Can we change our OpenMRN away from our usage of keyword "DISPLAY", or can we eliminate this from the ESP32 namespace?

Reply:
its worse than namespace pollution, on the arduino-esp32 side it is a #define, I had opted originally to rename the two references to DISPLAY in this file but it didn't seem right. I also don't like this option either.